Dockerfile

```
@@ -4,7 +4,7 @@ FROM ubuntu:focal as app
 
 RUN apt-get update && \
     apt-get upgrade -qy && apt-get install language-pack-en locales git \
```

```suggestion
    apt-get upgrade -qy && DEBIAN_FRONTEND=noninteractive apt-get install language-pack-en locales git \
```
Otherwise the docker build will ask for input and hang. (See notes in https://github.com/edx/edx-arch-experiments/issues/349)

---

Actually, passing `--no-install-recommends` to apt-get install might be even better, since tzdata was only a _recommended_ transitive dependency. If it works, it's a good idea, since it will shrink your image sizes. But there's some chance you're depending on a recommended package without realizing it; up to you whether the risk is worth it.

---

I had success building the image with `DEBIAN_FRONTEND=noninteractive`. The problem is, I am not sure how to test these image. I do not want to take change on thing I can't test.

---

Yeah, `DEBIAN_FRONTEND=noninteractive` should be safe. If xqueue is not something you can readily test, it's probably not worth shrinking the image size.